crit-go/magic-gen/Makefile

```diff
@@ -0,0 +1,53 @@
+GO ?= go
+CC ?= gcc
+ifeq ($(GOPATH),)
+export GOPATH := $(shell $(GO) env GOPATH)
+endif
+FIRST_GOPATH := $(firstword $(subst :, ,$(GOPATH)))
+GOBIN := $(shell $(GO) env GOBIN)
+ifeq ($(GOBIN),)
+	GOBIN := $(FIRST_GOPATH)/bin
+endif
+
+all: build test magicgen
+
+lint: 
+	@golint magic-gen.go
+
+build:
+	 @$(GO) build -v
+
+test/magicgen:
+	test_input=`mktemp`
+	test_output=`mktemp`
```

Comment:
Remove the two mktemp lines as you are not using the variables below anyway. You are hard-coding the input and output to `test_input` and `test_output`, which is probably okay here.